Core/src/org/sleuthkit/autopsy/casemodule/UnpackagePortableCaseProgressDialog.java

```diff
@@ -261,6 +292,33 @@ private File locate7ZipExecutable() {
 
             return exeFile;
         }
+
+        /**
+         * Check to see if the case has been unpacked before
+         */
+        private String hasCaseBeenUnpackedBefore(String packedCaseName) {
```

Comment:
Agree on the renaming and we've been trying to return Optional more instead of 'null' to prevent NPEs.

Reply:
Return empty string and check for it